cloud: add TiDB Cloud Dedicated Data Migration maintenance notification (#16863)
pingcap/docs: tidb-cloud/notification-2024-04-11-dm-feature-maintenance.md

---

> title: 2024-04-11 TiDB Cloud Dedicated Data Migration (DM) Feature Maintenance Notification
> summary: Learn about the details of TiDB Cloud Dedicated Data Migration (DM) feature maintenance on April 11, 2024, such as the maintenance window and impact.
> ---
> 
> # [2024-04-11] TiDB Cloud Dedicated Data Migration (DM) Feature Maintenance Notification

Reviewer: ```suggestion
title: 2024-04-11 TiDB Cloud Data Migration (DM) Feature Maintenance Notification
summary: Learn about the details of TiDB Cloud Data Migration (DM) feature maintenance on April 11, 2024, such as the maintenance window and impact.
---

# [2024-04-11] TiDB Cloud Data Migration (DM) Feature Maintenance Notification
```

This notification describes the details that you need to know about the maintenance for [Data Migration (DM) feature](/tidb-cloud/migrate-from-mysql-using-data-migration.md) of TiDB Cloud Dedicated on April 11, 2024.

## Maintenance window

- Start time: 2024-04-11 08:00 (UTC+0)
- End time: 2024-04-11 09:00 (UTC+0)
- Duration: 1 hour

## Impact

During the maintenance window, the DM feature for TiDB Dedicated clusters in the following regions will be affected:

- Cloud provider: AWS, region: Oregon (us-west-2)
- Cloud provider: AWS, region: N. Virginia (us-east-1)
- Cloud provider: AWS, region: Singapore (ap-southeast-1)
- Cloud provider: AWS, region: Seoul (ap-northeast-2)
- Cloud provider: AWS, region: Frankfurt (eu-central-1)
- Cloud provider: AWS, region: São Paulo (sa-east-1)
- Cloud provider: AWS, region: Oregon (us-west-2)
- Cloud provider: Google Cloud, region: Oregon (us-west1)
- Cloud provider: Google Cloud, region: Tokyo (asia-northeast1)
- Cloud provider: Google Cloud, region: Singapore (asia-southeast1)

> Apart from the DM feature in the TiDB cluster, all other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.

Reviewer: ```suggestion
The maintenance only affects the DM feature in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
```
